contracts/EthStorageContract.sol

```diff
@@ -30,6 +30,7 @@ contract EthStorageContract is StorageContract, Decoder {
         address _treasury,
         address _owner
     ) public payable initializer {
+        require(msg.value >= prepaidAmount, "prepaid amount is not enough");
```

Comment:
`prepaidAmount` is not set value yet, is it `_prepaidAmount` that we should check? 
Maybe a better place is inside `__init_storage` in `StorageContract`?